docs/Gemfile.lock

```diff
@@ -63,10 +63,9 @@ GEM
     rexml (3.2.5)
     rouge (4.1.0)
     safe_yaml (1.0.5)
-    sass-embedded (1.58.3-arm64-darwin)
-      google-protobuf (~> 3.21)
-    sass-embedded (1.58.3-x86_64-linux-gnu)
+    sass-embedded (1.59.2)
       google-protobuf (~> 3.21)
+      rake (>= 10.0.0)
```

Comment:
Same here, It would be good to know the reason for the change to the feature you have developed in this PR.

Reply:
I removed this to maintain it same as main branch.